packages/grafana-ui/src/components/Stack/VerticalStack.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { ThemeSpacingTokens } from '@grafana/data';
+
+import { Stack } from './Stack';
+
+export const VerticalStack = React.forwardRef<HTMLDivElement, React.PropsWithChildren<{ gap: ThemeSpacingTokens }>>(
```

Comment:
Seeing that `Stack` already has vertical direction by default, is this component still necessary?

Reply:
Good point 🤔 👍